utils/tracing/integ-tests/client-server/goclient/Dockerfile

```diff
@@ -27,6 +27,7 @@ RUN apk add --no-cache make
 COPY ./utils/tracing/integ-tests/client-server/goclient ./
 COPY ./utils/tracing/go ./utils/tracing/go
 RUN mkdir /log
+RUN go mod edit -replace github.com/ease-lab/vSwarm/utils/tracing/go=./utils/tracing/go
```

Comment:
Why dont you have the replace in the go.mod file?
With this the docker file uses the local updates but if you test it without docker hence simply `go build` it will pull instead of taking the local files right?

Reply:
The folder structure is being changed in the Dockerfile. Specifically `COPY ./utils/tracing/integ-tests/client-server/goclient ./`. If we remove any replace line for tracing that would throw an error because tracing is currently not merged into main of vSwarm.

Reply:
Because the updated tracing module name is in this PR.

Reply:
this is exactly why it should be in a separate PR.

Reply:
Yes but it will take unnecessary time to separate the changes again. We can create issues for the changes that need to be made, merge this branch, and then fix those issues. That will be much faster and simpler.

Reply:
I disagree. this PR is called "relay" so everything else should be in the corresponding PRs.

Reply:
But we talked about this in the meeting and it was decided that we can merge this PR for now and subsequently the PRs would be function-wise.

Reply:
So first I agree with @harshitgarg22 that this was discussed in https://github.com/ease-lab/vSwarm/pull/64#issuecomment-1102343921

Reply:
Anyway. Pointing with the fingers to other does not help now to make progress.
In that case Ill reopen the PR for tracing merge rebase and merge it. Then tracing is fixed.
Harshit will update the relay stuff accordingly

Reply:
Indeed, I agreed to merge this PR before looking at its contents, it was my fault. Now, it is clear to me that we are unable to verify all of its contributions carefully. Moreover, this PR brings stuff that requires changes after this PR to correct it, which is unacceptable. Plus some tests seem to be broken by this PR, [example](https://github.com/ease-lab/vSwarm/runs/6111412939?check_suite_focus=true). 

It is better to go slower than waste time in debugging various issues.

Reply:
Ok lets do it. I rebased the tracing on the most recent commits in the main and merged the PR.
@harshitgarg22 please to ahead and also rebase this PR to the current state.
Then we can see what still breaks and go over it step by step